src/lib/logical_query_plan/abstract_lqp_node.cpp

```diff
@@ -167,8 +167,8 @@ std::vector<LQPOutputRelation> AbstractLQPNode::output_relations() const {
 
 size_t AbstractLQPNode::output_count() const { return _outputs.size(); }
 
-std::shared_ptr<AbstractLQPNode> AbstractLQPNode::deep_copy(LQPNodeMapping node_mapping) const {
-  return _deep_copy_impl(node_mapping);
+std::shared_ptr<AbstractLQPNode> AbstractLQPNode::deep_copy(LQPNodeMapping extern_node_mapping) const {
```

Comment:
"extern"... this is the node mapping of all nodes below this node... so maybe "input_node_mapping"?

Reply:
Not my name - that already existed in the hpp. I can change it though.